walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,72 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Collections;
+import com.google.sps.Event;
+import com.google.sps.TimeRange;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Set;
+import java.util.Iterator;
+import java.util.ArrayList;
+import java.util.Arrays;
 
 public final class FindMeetingQuery {
+  private boolean eventParticipantInMeeting(Set<String> eventAttendees, 
+                                              Collection<String> meetingAttendees) {
+    for (String eventAttendee : eventAttendees) {
+      if (meetingAttendees.contains(eventAttendee)) {
+        return true;
+      }
+    }
+    return false;
+  }
```

Comment:
consider to use set.retainAll() function to do this intersection

Reply:
This is a pretty cool function! Current implementation uses this now.